samples/README.md

```diff
@@ -0,0 +1,65 @@
+# Pub/Sub Lite Spark Connector Word Count Samples
+
+This directory contains a word count sample for Pub/Sub Lite Spark Connector.
+
+## Authentication
+
+Please see the [Google cloud authentication guide](https://cloud.google.com/docs/authentication/). 
+The recommended approach is to use Application Default Credentials by setting `GOOGLE_APPLICATION_CREDENTIALS`.
+
+## Environment Variables
+Set the following environment variables:
+```
+PROJECT_ID=your-project-id
+REGION=us-central1 # or your region
+ZONE_ID=b # or your zone id
+TOPIC_ID=test-topic # or your topic id to create
+SUBSCRIPTION_ID=test-subscrciption # or your subscription to create
+PARTITIONS=1 # or your number of partitions to create
+CLUSTER_NAME=waprin-spark7 # or your Dataproc cluster name to create
+BUCKET=gs://your-gcs-bucket
+SUBSCRIPTION_PATH=projects/$PROJECT_NUMBER/locations/$REGION-$ZONE_ID/subscriptions/$SUBSCRIPTION_ID
```

Comment:
```suggestion
SUBSCRIPTION_PATH=projects/$PROJECT_ID/locations/$REGION-$ZONE_ID/subscriptions/$SUBSCRIPTION_ID
```

Reply:
I think this needs a project number, I added that env variable above.

Reply:
I gave it a try using PROJECT_ID and it seemed to have worked.

Reply:
hm I think we added this support recently. Anyway I switched back to project number in all readme user samples.